File: code/48103.py
import math
a, b, c, d = [int(x) for x in input().split()]
def hd(x, y):
  tam = x
  x = y 
  y = tam
if a> b:
  hd(a,b)
if b> c:
  hd(b,c)
if c > d:
  c = d 
k = max(a, max(b,c))
print(k)

File: code/48103.pas
uses math;
var a, b, c, d, k : integer;
procedure hoandoi(var a, b : integer);
var x : integer;
begin
    x:=a; 
    a:=b;
    b:=x;
end;
begin
    readln(a, b, c, d);
    if a>b then hoandoi(a,b);
    if b>c then hoandoi(b,c);
    if c>d then c:=d;
    k:=max(a, max(b,c));
    write(k)
end.
